apps/site/components/ImageGallery/ImageGalleryUsage.tsx

```diff
@@ -9,9 +9,10 @@ const ImageComponent = ({ url, alternateName, onLoad }) => (
 
 export interface ImageGalleryUsageProps {
   images?: ImageElementData[]
+  imagePos?: "top" | "center" | "bottom"
```

Comment:
I prefer calling it `imagePosition`, we have bigger props than that 😬

Reply:
Me too!